simple_https_server/src/tls13server.rs

```diff
@@ -42,6 +42,8 @@ struct Cli {
     ///   * SHA384_Aes256Gcm_EcdsaSecp256r1Sha256_X25519
     ///   * SHA384_Aes256Gcm_RsaPssRsaSha256_P256
     ///   * SHA384_Aes256Gcm_RsaPssRsaSha256_X25519
+    ///   * SHA256_Chacha20Poly1305_EcdsaSecp256r1Sha256_X25519Kyber768Draft00
+    ///   * SHA256_Chacha20Poly1305_EcdsaSecp256r1Sha256_X25519MlKEM76
```

Comment:
I think all of these comments have the same typo: missing 8 at the end and lower case l.